kuka_controllers/src/joint_impedance_controller.cpp

```diff
@@ -83,11 +135,11 @@ controller_interface::return_type JointImpedanceController::update(
   const rclcpp::Duration &)
 {
 
-  for (auto index = 0; index < 6; ++index) {
```

Comment:
if you change the order of the interfaces here, you should also change it in the hardware interface, otherwise they will be mixed up

Reply:
Is this order better though? I personally prefer the other one, since you can tell at a glance if a parameter is damping or stiffness based on which half of the vector they are in. Probably not important enough to address right now though.

Reply:
In the joint trajectory controller it was created in this order, thats why i did in this way.

Reply:
This is connected to the changes in rox_hardwer_interface.cpp